bitbots_bringup/scripts/launch_on_remote.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+##########
+# Params #
+##########
+# 1 param is the remote device
+# 2 param is the ros master
+# 3 param is the ros_package
+# 4 param is the launch file
+# Following params are the roslaunch arguments
+
+ssh -t -t $1 'source ~/.zshrc; export ROS_MASTER_URI=http://'$2':11311/; zsh -o HUP -c "roslaunch '${@:3}'"'
```

Comment:
As codefactor sais, please use `$3` instead of `${@:3}`

Reply:
> As codefactor sais, please use $3 instead of ${@:3}

But this is not the same thing. I want a list of all params after the third, not only the third.

The behavior should look like this:
```
./launch_on_remote.sh a b c d e f
source ~/.zshrc; export ROS_MASTER_URI=http://b:11311/; zsh -o HUP -c "roslaunch c d e f"
```
Not like this:
```
./launch_on_remote.sh a b c d ef
source ~/.zshrc; export ROS_MASTER_URI=http://b:11311/; zsh -o HUP -c "roslaunch c"
```
I think the solution that code factor wants is this `${*:3}`.